script.js

```diff
@@ -1,104 +1,56 @@
-/* eslint-disable max-classes-per-file */
-
-const form = document.querySelector('.form');
-const booksDiv = document.querySelector('.books');
-const addNew = document.querySelector('.add-book');
-const viewList = document.querySelector('.my-grid');
-const viewContact = document.querySelector('.contact-info');
-const allNavLinks = document.querySelectorAll('.load');
-const listLink = document.querySelectorAll('.link-1');
-const AddLink = document.querySelectorAll('.link-2');
-const contactLink = document.querySelectorAll('.link-3');
-const dateContainer = document.querySelector('.date');
-const date = new Date().toLocaleDateString('en-us', { month: 'long', day: '2-digit', year: 'numeric' });
-const time = new Date().toLocaleTimeString('en-us', { hour: 'numeric', minute: '2-digit' });
-const dateTime = `${date} ${time}`;
-
-dateContainer.innerHTML = dateTime;
-
-listLink.forEach((link) => link.addEventListener('click', () => {
-  viewList.style.display = 'flex';
-  addNew.style.display = 'none';
-  viewContact.style.display = 'none';
-}));
-
-AddLink.forEach((link) => link.addEventListener('click', () => {
-  addNew.style.display = 'flex';
-  viewList.style.display = 'none';
-  viewContact.style.display = 'none';
-}));
-
-contactLink.forEach((link) => link.addEventListener('click', () => {
-  viewContact.style.display = 'flex';
-  viewList.style.display = 'none';
-  addNew.style.display = 'none';
-}));
-
 class Books {
-  constructor(title, author, id) {
+  constructor(title, author) {
     this.title = title;
     this.author = author;
-    this.id = id;
   }
 }
 
```

Comment:
- [ ] According to the project requirement you should create a collection that keeps a list of books. For example: you can use array of object for that and Display all books saved in the collection in the top part of the page.
- [ ] [Optional] In the upcoming project you will use class so I think for now you can use plain javascript function.
### Requirment
![his](https://github.com/kkmanuu/Awesome-Book/assets/81727988/4f7e1e01-d05b-46f6-b59a-96a828bd591d)

Reply:
@roshan-bajgain Done Making changes